pages/download.py

```diff
@@ -111,16 +111,14 @@ def get_content(self):
 """)))+div("macos",md(_("""
 ### OS X
 
-[Install Freenet 0.7](assets/jnlp/freenet.jnlp) using Java Web Start.  
-This requires that Java is installed and that Java Web Start is enabled. If
-it doesn't work, try [installing Java](http://www.java.com/),
-then downloading [the installer](assets/jnlp/freenet_installer.jar) and
-opening it. It might take a moment to launch.
-
-**Note:** Mavericks does not ship with Java Web Start enabled. We would like
-to make a _.dmg_ for easier OS X installation but don't know how yet. If you
-are a developer and would like to join us and help it would be much
-appreciated!
+Download and run [the installer](https://github.com/steveatinfincia/mactray/releases/download/v2.0.0/FreenetTray_2.0.0-signed.zip)
+([gpg signature](https://github.com/steveatinfincia/mactray/releases/download/v2.0.0/FreenetTray_2.0.0-signed.zip.sig))
```

Comment:
Perhaps not as part of this changeset, but it'd be nice to link to instructions on verifying signatures like [Tails does](https://tails.boum.org/download/index.en.html#index3h1).